services/wfm/src/main/java/org/openkilda/wfm/topology/flow/validation/FlowValidator.java

```diff
@@ -113,4 +122,41 @@ void checkFlowForEndpointConflicts(Flow requestedFlow) throws FlowValidationExce
                             conflictedFlow.get().getFlowId()));
         }
     }
+
+    /**
+     * Validates switch existing.
```

Comment:
`Ensure switches are exists.` is more accurate comment.

Reply:
Resolved